src/mmw/js/src/user/views.js

```diff
@@ -279,8 +281,93 @@ var ForgotModalView = Marionette.ItemView.extend({
     }
 });
 
+var ItsiRegisterModalView = Marionette.ItemView.extend({
+    template: itsiRegisterModalTmpl,
+
+    className: 'modal modal-large fade',
+
+    ui: {
+        'registerButton': '#itsi-register-button',
+        'username': '#username',
+        'first_name': '#first-name',
+        'last_name': '#last-name',
+        'agreed': '#agreed',
+        'registerModal': '#itsi-register-modal'
+    },
+
+    events: {
+        'click @ui.registerButton': 'validate',
+        'keyup @ui.registerModal': 'handleKeyUpEvent'
+    },
+
+    initialize: function(options) {
+        this.app = options.app;
```

Comment:
Marionette views have a helper function called [mergeOptions](http://marionettejs.com/docs/v2.4.1/marionette.view.html#viewmergeoptions) that we may be able to utilize here.